src/utils.lisp

```diff
@@ -146,6 +147,34 @@ ELSE will be executed."
                      ,@progn)
                    (list-match-case ,tgt ,@others))))))
       nil))
+|#
+
+(defun list-match-pattern->trivia (pattern)
```

Comment:
Does this make a dependency on trivia?

Reply:
Yes, it does. I looked at `list-match-case ` and it was too much trouble to figure it out to fix a single case statement.

Reply:
OK. I'm not the maintainer of fiveam.
But there are Cl implementations that don't work well with trivia, like ABCL.
fiveam should work with the largest set of CL implementations.

Reply:
Is there a workaround to not using Trivia but make your `set-equa` usecase work?

Reply:
@mdbergmann 
> OK. I'm not the maintainer of fiveam. But there are Cl implementations that don't work well with trivia, like ABCL. fiveam should work with the largest set of CL implementations.

I did not know about that. Do you have any pointers to information about trivia's (in)compatibility with various lisp implementations?

I agree with you that 5AM should aim to be as portable as possible.

Reply:
@mdbergmann I believe that the best work-around -- although it is not good -- would be simply to rip out the use of pattern-matching and have a hand-written `cond` expression with `destructuring-bind`.

As I implied earlier, there's a lot more code to make `list-match-case` work than there is actual use of this macro!

It would be nice if 5AM had some github-actions to run the tests on multiple lisp implementations (like we have for ASDF and ITERATE on common-lisp.net).

Reply:
I just tried on ABCL. Trivia can be compiled and basic pattern matching works.
But running the whole test suite for some reason fails, though for some other reason than trivia.